src/services/web-monitoring-db.js

```diff
@@ -232,6 +232,7 @@ export default class WebMonitoringDb {
   getDiff (pageId, aId, bId, diffType) {
     return fetch(this._createUrl(`pages/${pageId}/changes/${aId}..${bId}/diff/${diffType}`, {format: 'json'}))
       .then(response => response.json())
+      .then(throwErrorResponse('Could not load diff'))
```

Comment:
The way that the function is named made me think that we were always throwing an error here, which is definitely not the case. I understand we're checking for errors, but this threw me off at first.

Reply:
How about `throwIfErrorResponse` or `throwIfError`?

Reply:
+1 `throwIfError`